### PR TITLE
Jb50531

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
The very first 4 commits belong to PR #101

File: buildqtc.sh [hunk 1]
```diff
@@ -340,6 +340,24 @@ build_unix_qtc() {
 
 	make deployqt
 
+    # Deploy QtWebEngine
+    if [[ $UNAME_SYSTEM == "Linux" ]]; then
+        mkdir -p "$QTC_INSTALL_ROOT/lib/Qt/libexec"
+        cp "$OPT_QTDIR/qtbase/libexec/QtWebEngineProcess" "$QTC_INSTALL_ROOT/lib/Qt/libexec/"
```
Comment (vigejolla) [inline buildqtc.sh:346]: I thought the jury is still out pondering whether we should use WebEngine or not?

File: defaults.sh [hunk 2]
```diff
@@ -50,10 +50,10 @@
 DEF_WIN_ICU_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/icu/icu4c-68_2-Win64-MSVC2019.zip"
 
 # The download URL for the Windows LLVM/Clang binaries
-DEF_WIN_LLVM_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/llvm/libclang-release_80-based-windows-vs2017_32.7z"
+DEF_WIN_LLVM_DOWNLOAD_URL="$DEF_URL_PREFIX/win32-binary-artifacts/llvm/libclang-release_110-based-windows-vs2019_64.7z"
```
Comment (vigejolla) [inline defaults.sh:53]: Perhaps we should make a clean break here and move the artifacts from "win32-binary-artifacts" to "win64-binary-artifacts"?

File: buildqtc.sh (hunk 1 above)
Comment (martyone) [reply buildqtc.sh:346]: yup, but I would anyway commit it and revert with later commits in order to have it preserved in the history - it was not trivial to achieve.

File: defaults.sh (hunk 2 above)
Comment (martyone) [reply defaults.sh:53]: I thought so too, but there is nothing like win64. The API is still called win32, no matter of the word size. Obviously we are not alone who did not realize, so there are some artifacts with win64 in filename IIRC.

File: buildqtc.sh (hunk 1 above)
Comment (vigejolla) [reply buildqtc.sh:346]: Right, and we already have the task in bz to change the documentation. That task can be easily changed to change this instead :)

File: defaults.sh (hunk 2 above)
Comment (vigejolla) [reply defaults.sh:53]: The API is no longer called win32, they renamed it to "Windows API" at some point. Even though the term win32 seems to still be there in e.g. the URL: https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-api-list. It's not a big deal, but it just feels wrong to have 64 binaries in a directory with "32" in the name.

**Review by vigejolla — APPROVED** [review]
LGTM

File: defaults.sh (hunk 2 above)
Comment (martyone) [reply defaults.sh:53]: OK, let's change it to just `win` next time we touch this? (I'd like to get this off the table finally for this round)

**Pull request merged**